agents/locobot/teleop.py

```diff
@@ -231,78 +270,83 @@ def test_command(sid, commands, yaw_velocity):
                                                        cone_height = .4,
                                                        resolution=20)
         robot_orientation.compute_vertex_normals()
-        robot_orientation.paint_uniform_color([0.1, 0.9, 0.1])
+        robot_orientation.paint_uniform_color([1.0, 0.5, 0.1])
         
-        robot_orientation.translate([y, -x, 0.5], relative=False)
+        robot_orientation.translate([y, -x, 0.], relative=False)
         robot_orientation.rotate(o3d.geometry.get_rotation_matrix_from_axis_angle([0, math.pi/2, 0]))
-        robot_orientation.rotate(o3d.geometry.get_rotation_matrix_from_axis_angle([0, 0, yaw]))        
+        if yaw != 0:
+            robot_orientation.rotate(o3d.geometry.get_rotation_matrix_from_axis_angle([0, 0, yaw]))        
```

Comment:
what are the two rotations for here? also, do we need the if check for the second rotation?

Reply:
will add comments, but this is the cylinder representating the orientation of the robot, so it needs to be first made parallel to the floor, and then rotated by `yaw`